### PR TITLE
Fix having to remove the cache directories after clearing the cache

### PR DESCRIPTION
**Pull request by carakas**
## Type

- Enhancement

## Resolves the following issues

Sometimes after clearing cache or preparing for reinstall I still get errors

## Pull request description

I usually remove the folders in the cache directory, after that the problem is fixed

I'm not sure if it has something to do with my setup (vagrant) so I want to know if you also have this problem and if this fixes it for you

### CODE REVIEW

**Comment by jessedobbelaere** [comment]
Yeah won't hurt to add this I think

File: tools/remove_cache [hunk 1]
```diff
@@ -21,5 +21,6 @@ rm -f `find ${SCRIPT_PATH}/../app/cache/ ! -name ".gitignore" -type f ! -path *.
 
 ${SCRIPT_PATH}/../app/console cache:clear
 ${SCRIPT_PATH}/../app/console cache:clear --env=prod
+${SCRIPT_PATH}/../app/console cache:clear --env=install
```
Comment (WouterSioen) [inline tools/remove_cache]: I would even dare to add `rm -rf ${SCRIPT_PATH}/../app/cache/*`

File: tools/prepare_for_reinstall [hunk 2]
```diff
@@ -21,6 +21,10 @@ clear_file_caches() {
 
     rm -f `find ${SCRIPT_PATH}/../src/Backend/Cache/Locale/ ! -name ".gitignore" -type f ! -path *.svn/* -type f`
     rm -f `find ${SCRIPT_PATH}/../src/Backend/Cache/CompiledTemplates/ ! -name ".gitignore" -type f ! -path *.svn/* -type f`
+
+    rm -rf `find ${SCRIPT_PATH}/../src/Backend/Cache/dev/`
+    rm -rf `find ${SCRIPT_PATH}/../src/Backend/Cache/install/`
+    rm -rf `find ${SCRIPT_PATH}/../src/Backend/Cache/prod/`
```
Comment (WouterSioen) [inline tools/prepare_for_reinstall]: Are these new paths? I can't remember having these in `src/Backend/Cache`?

Comment (carakas) [inline tools/prepare_for_reinstall]: hmm I think it should be app and that is was a copy mistake

File: tools/remove_cache (hunk 1 above)
Comment (carakas) [inline tools/remove_cache]: fixed in 03c25d921812d7f1d54f50a4725f9002d3da301f

File: tools/prepare_for_reinstall (hunk 2 above)
Comment (carakas) [inline tools/prepare_for_reinstall]: fixed in 03c25d921812d7f1d54f50a4725f9002d3da301f

**Comment by carakas** [comment]
fixes #1695

**Pull request merged**